dgsudharsan/sonic-buildimage
# Support for Mellanox-SN2700 based SKU for docker-sonic-vs

#### Why I did it
Added support for Mellanox-SN2700 based SKU for docker-sonic-vs

#### How I did it
Copied relevant platform files. Modified orchagent scripts to infer platform from hwsku.

#### How to verify it
Run the tests that depend on platform type in vs and ensure they pass using the new approach.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
